### PR TITLE
fix chrome pc.sctp in plan-b mode

### PR DESCRIPTION
**Pull request by fippo**
tentative fix

### CODE REVIEW

File: src/js/common_shim.js [hunk 1]
```diff
@@ -164,6 +164,20 @@ export function shimMaxMessageSize(window) {
       window.RTCPeerConnection.prototype.setRemoteDescription;
   window.RTCPeerConnection.prototype.setRemoteDescription = function() {
     this._sctp = null;
+    // Chrome unilaterally decided to not expose .scpt in plan-b mode.
+    // As usual, adapter.js has to do the 'ugly hack' to cover up the mess.
+    if (browserDetails === 'chrome' && ('sctp' in this)) {
```
Comment (dagingaa) [inline src/js/common_shim.js]: Should this be negated for sctp?

```suggestion
    if (browserDetails === 'chrome' && !('sctp' in this)) {
```

Comment (fippo) [reply src/js/common_shim.js]: the point is to do this once, even in plan-b mode. But it doesn't work as intended...

File: src/js/common_shim.js [hunk 2]
```diff
@@ -164,6 +164,21 @@ export function shimMaxMessageSize(window) {
       window.RTCPeerConnection.prototype.setRemoteDescription;
   window.RTCPeerConnection.prototype.setRemoteDescription = function() {
     this._sctp = null;
+    // Chrome unilaterally decided to not expose .scpt in plan-b mode.
```
Comment (lgrahl) [inline src/js/common_shim.js]:
```suggestion
    // Chrome unilaterally decided to not expose .sctp in plan-b mode.
```

**Review by lgrahl — DISMISSED** [review]
IIRC this won't work since https://github.com/webrtcHacks/adapter/pull/979/files#diff-26e41bf001b9f7daa490e681199d92edR67 returns early.

The code should be the same for plan B. Only the above mentioned check needs to be changed to look for plan B... since you only know that once local/remote description is being applied, ugh... it will be ugly.

**Comment by fippo** [comment]
Now there is an odd failure in chrome beta without experimental web platform features when sending a too-large message. This is a Chrome bug and seems to be fixed in Canary.

File: src/js/common_shim.js [hunk 3]
```diff
@@ -164,6 +164,21 @@ export function shimMaxMessageSize(window) {
       window.RTCPeerConnection.prototype.setRemoteDescription;
   window.RTCPeerConnection.prototype.setRemoteDescription = function() {
     this._sctp = null;
+    // Chrome unilaterally decided to not expose .sctp in plan-b mode.
+    // As usual, adapter.js has to do an 'ugly worakaround '
```
Comment (fippo) [inline src/js/common_shim.js]: nit: extra space

**Review by lgrahl — APPROVED** [review]
Since Plan-B can only be enabled locally, should be fine now.

**Pull request merged**